Object/People.pm

```diff
@@ -126,6 +139,22 @@ Get date of death.
 
 Returns string.
 
+=head2 C<work_period_start>
+
+ my work_period_start = $obj->work_period_start;
```

Comment:
Mělo by být ```my $work_period_start = $obj->work_period_start;```
Chybějící dolar. A to stejné by mělo být v SYNOPSIS sekci.

Reply:
nějak se mi to vytratilo, opraveno